domain/personallearningplan/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/personallearningplan/Exceptions.kt

```diff
@@ -40,6 +41,9 @@ class InvalidGoalStateException(
 ) :
   RuntimeException("Could not ${action.name.lowercase()} goal with reference [$goalReference] for prisoner [$prisonNumber]: Goal was in state [$status] that can't be ${action.name.lowercase()}d")
 
+class ActiveGoalStepsException(val goalReference: UUID, val prisonNumber: String) :
```

Comment:
Maybe a poor name choice but couldn't think of anything better - Exception to be raised when an attempt is made to complete a goal but it has active child steps.